source/storing-data/provenance.rst

```diff
@@ -2,3 +2,213 @@ Decentralized Provenance Tracking
 =================================
 .. contents::
    :local:
+
+QIIME 2 provides automatic, integrated, and decentralized tracking of analysis
+metadata, including information about the host system, the computing
+environment, Actions performed, parameters passed, primary sources cited, and
+more. We describe all of this information about "how an analysis was
+performed" as "provenance".
+
+The notion of a QIIME 2 :term:`Result` is central here. Whenever an
+:term:`Action` is performed on some data with QIIME 2, the framework
+captures relevant metadata about the action and environment and stores it in
+the Action's Result. When/if that Result is saved as an :term:`Archive`, the
+captured provenance data is stored within the Archive as well.
+:ref:`provenance-structure` contains a detailed discussion of the
+file structure which holds provenance metadata.
+
+.. note::
+   *When* Results are saved as Archives is interface-defined.
+   Results are saved automatically by ``q2cli``, every time a user runs a command.
+   They must be saved manually by users of the `Artifact API <https://docs.qiime2.org/2021.4/interfaces/artifact-api/>`_.
+   This allows API uses to reduce I/O, and keeps things simpler for CLI users.
+
+Why Capture Provenance Data?
+----------------------------
+
+Provenance gives QIIME 2 users, paper reviewers, and developers valuable
+tools for zero-cost documentation, study validation and reproduction,
+workflow pipelining, and software maintenance and repair.
+
+Among the benefits of this model are:
+
+- Analyses are *fully* reproducible.
+- Analyses self-document, reducing the need for investigator notetaking. For example, `q2view <https://view.qiime2.org/>`_ produces directed provenance graphs. QIIME 2 Artifacts bring their citations with them. Methods-section text could theoretically be generated from a collection of QIIME 2 Artifacts.
```

Comment:
Using the term `retrospective provenance` may be useful here, although you would have to find a way to work it into the text naturally. This term is helpful because it is distinct from prospective provenance (which would be like a script).

Reply:
Useful distinction, but it didn't fit nicely so I addressed this with a paragraph in the intro.